cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js

```diff
@@ -17,9 +17,9 @@ Then('the default working list should be displayed', () => {
         '55 Male',
         '26 Female',
         '35 Male',
+        '12 Male',
```

Comment:
hey @simonadomnisoru , changing the order may still remain flaky tests, in newer version we did the [grouping by date ](https://github.com/dhis2/capture-app/blob/master/cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js#L13), can you have a look

Reply:
Hey, I updated the cypress scenarios just like in the newer versions. Thank you for the suggestion!